osu.Game/Beatmaps/BeatmapProcessor.cs

```diff
@@ -11,11 +11,27 @@ public interface IBeatmapProcessor
         IBeatmap Beatmap { get; }
```

Comment:
let's move this to its own file already

Reply:
and add an interface level comment explaining why you'd want a beatmap processor in the first place (since it seems optional).